clang/test/CodeGenSYCL/sycl-framework-metadata1.cpp

```diff
@@ -1,7 +1,9 @@
 // Test checks that !sycl-framework metadata is emitted only to functions
 // whose topmost namespace is sycl.
 
-// RUN: %clang_cc1 -fsycl-is-device -triple spir64-unknown-unknown -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -emit-llvm %s -o - | FileCheck %s
```

Comment:
I think @elizabethandrews meant to also use the single_task interface from the mock header instead of the one defined below for this.